openstack_controller/metadata.csv

```diff
@@ -1,59 +1,209 @@
+openstack.nova.vcpus,gauge,,cpu,,Number of vCPUs available on this hypervisor host,0,openstack_controller,nova vcpus,
+openstack.nova.vcpus_used,gauge,,cpu,,Number of vCPUS used on this hypervisor host,0,openstack_controller,nova vcpus used,
 openstack.nova.service.up,gauge,,,,"Reports a value of 1 if this Nova compute service is up, 0 otherwise",0,openstack_controller,nova service up,
+openstack.ironic.response_time,gauge,,millisecond,,"The response time of the Ironic API in milliseconds",0,openstack_controller,ironic response,
 openstack.ironic.conductor.up,gauge,,,,"Reports a value of 1 if this conductor is alive, 0 otherwise",0,openstack_controller,ironic conductors,
 openstack.ironic.node.up,gauge,,node,,Number of Bare Metal nodes that are not in maintenance mode,0,openstack_controller,ironic nodes ct,
 openstack.ironic.node.count,gauge,,node,,Number of Bare Metal nodes,0,openstack_controller,ironic nodes up,
-openstack.keystone.response_time,gauge,,,,Time it takes for the keystone component to respond to a basic request,0,openstack_controller,keystone response time,
+openstack.keystone.response_time,gauge,,millisecond,,Time it takes for the keystone component to respond to a basic request,0,openstack_controller,keystone response time,
 openstack.keystone.domains.count,gauge,,,,Number of domains,0,openstack_controller,keystone domains count,
+openstack.keystone.domains.enabled,gauge,,,,Number of domains enabled,0,openstack_controller,keystone domains enabled,
+openstack.keystone.groups.count,gauge,,,,Number of groups,0,openstack_controller,keystone groups,
+openstack.keystone.groups.users,gauge,,user,,"Number of group users, tagged by group.",0,openstack_controller,keystone groups users,
+openstack.keystone.limits,gauge,,,,"Override limit for a given resource",0,openstack_controller,keystone limits,
+openstack.keystone.projects.count,gauge,,,,"Number of projects",0,openstack_controller,keystone projects,
+openstack.keystone.projects.enabled,gauge,,,,"Number of enabled projects",0,openstack_controller,keystone projects enabled,
+openstack.keystone.services.count,gauge,,service,,"Number of web services that the user can access through a URL",0,openstack_controller,keystone services,
+openstack.keystone.services.enabled,gauge,,service,,"Number of web services enabled that the user can access through a URL",0,openstack_controller,keystone services enabled,
+openstack.keystone.users.count,gauge,,user,,"Number of users",0,openstack_controller,keystone users,
+openstack.keystone.users.enabled,gauge,,user,,"Number of enabled users",0,openstack_controller,keystone users enabled,
+openstack.neutron.response_time,gauge,,millisecond,,"The response time of the Neutron API in milliseconds",0,openstack_controller,neutron response,
+openstack.neutron.agents.admin_state_up,gauge,,,,"The administrative state of the agent, which is 1 if up, and 0 otherwise",0,openstack_controller,neutron agents state,
```

Comment:
```suggestion
openstack.neutron.agents.admin_state_up,gauge,,,,"The administrative state of the agent, which is 1 if up, 0 otherwise",0,openstack_controller,neutron agents state,
```